modules/appeals_api/app/services/appeals_api/pdf_construction/higher_level_review/v2/form_fields.rb

```diff
@@ -184,6 +184,12 @@ def boxes
             issues_pg2: [].tap do |n|
               6.times { |i| n << { at: [-3, 675 - (46.5 * i)], width: 369, height: 43 } }
             end,
+            soc_date_pg1: [].tap do |n|
+              7.times { |i| n << { at: [375, 315 - (46.5 * i)], width: 160, height: 15 } }
```

Comment:
is the 6 and 7 captured in a constant on Structure?

Reply:
I'm just repeating the pattern found above for issues. I can get in there and make these constants.